### PR TITLE
Fixup explicitacquisition tests

### PR DESCRIPTION
**Pull request by jaroel**
Also see https://github.com/plone/Products.CMFPlone/pull/3855

### CODE REVIEW

**Comment by jaroel** [comment]
@jenkins-plone-org please run jobs

**Comment by jaroel** [comment]
@jenkins-plone-org please run jobs

File: src/plone/rest/tests/test_explicitacquisition.py [hunk 1]
```diff
@@ -11,18 +11,17 @@
 
 import unittest
 
-
 try:
-    from Products.CMFCore.interfaces import IShouldAllowAcquiredItemPublication
+    from Products.CMFCore.explicitacquisition import SKIP_PTA
```
Comment (tisto) [inline src/plone/rest/tests/test_explicitacquisition.py:15]: @jaroel I am wondering if we should use the abbreviation "SKIP_PTA" here. By just looking at the PR I have no clue what this is about. I'd need to look into CMFCore to understand what it means and does. Though, I am not sure if this is still possible to change without breaking too many things...

Comment (jaroel) [reply src/plone/rest/tests/test_explicitacquisition.py:15]: @tisto Don't think it'll be a problem. It's probably not used anywhere yet. CMFCore >= 3.2 is very recent and we made the PR.

File: src/plone/rest/tests/test_explicitacquisition.py [hunk 2]
```diff
@@ -53,5 +52,62 @@ def test_portal_foo(self):
 
     def test_portal_foo_acquired(self):
         self.traverse("/plone/foo/foo")
+        notify(PubAfterTraversal(self.request))
+
+
+@unittest.skipUnless(
+    SKIP_PTA is not None,
+    "We have Products.CMFPlone >= 3.2",
```
Comment (mauritsvanrees) [inline src/plone/rest/tests/test_explicitacquisition.py:60]:
```suggestion
    "We have Products.CMFCore >= 3.2",
```

**Comment by jaroel** [comment]
I've removed the references to PTA here.
https://jenkins.plone.org/job/pull-request-6.1-3.11/328/

**Review by mauritsvanrees — APPROVED** [review]
LGTM.
I did some small fixups, and started Jenkins jobs, also on 6.0 (where no CMFPlone branch should be needed).
Can be merged when green.

**Comment by mauritsvanrees** [comment]
All green!  (Except for the changelog one that we can ignore.)

**Pull request merged**